tests/log-out.test.js

```diff
@@ -0,0 +1,29 @@
+import LogoutPage from '../pageobjects/logout.page'
+import LoginPage from '../pageobjects/login.page'
+
+
+describe('Banking', () => {
+    it('Should logout', async () => {
+        await LoginPage.openWeb();
+
+        await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+
+        await LogoutPage.logOut();
+
+        await expect(await browser.getUrl()).to.contain('login?logout');
+    });
+
+    it('Should show the login screen when going back to the previous page after loging out', async () => {
+        await LoginPage.openWeb();
+
+        await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+
+        await LogoutPage.logOut();
+
+        await expect(await browser.getUrl()).to.contain('login?logout');
```

Comment:
This assert is not necessary within the context of this specific test, as you already have a test that validates this sub-flow. Besides that, this decreases adherence to the Single Responsability Principle, making your test lose focus. Consider minimizing and focalizing the assertions within a test (ideally limiting it to only one per test)